src/Files.App/UserControls/SideBar/SideBarView.xaml

```diff
@@ -128,24 +128,39 @@
 			x:Name="ContentPresenter"
 			Grid.Row="1"
 			Grid.Column="2"
-			Margin="2,0,0,0"
+			Margin="2,0,8,0"
 			HorizontalAlignment="Stretch">
 			<Grid.RowDefinitions>
+				<RowDefinition Height="Auto" />
 				<RowDefinition Height="*" />
 				<RowDefinition Height="Auto" />
 			</Grid.RowDefinitions>
 
+			<!--  Header  -->
+			<Border
+				x:Name="ContentHeaderPresenter"
+				Grid.Row="0"
+				Margin="0,0,0,4"
```

Comment:
This margin provides space between the header and content. I'm not so happy about this and I'm open to other ideas.